Adding open-source repositories. (#20)
tauri-apps/awesome-tauri: README.md

- [Mr Tagger](https://github.com/probablykasper/mr-tagger) - Music file tagging app.
- [Time Machine Inspector](https://github.com/probablykasper/time-machine-inspector) - Find out what's taking up your Time Machine backup space.
- [Identia](https://github.com/iohzrd/identia) - Decentralized social media on IPFS.

> - [Mail-Dev](https://github.com/samirdjelal/mail-dev) - Cross-platform, Local SMTP Server For Email Testing/Debugging.

Reviewer: ```suggestion
- [Mail-Dev](https://github.com/samirdjelal/mail-dev) - Cross-platform, local SMTP server for email testing/debugging.
```

> - [Bidirectional](https://github.com/samirdjelal/bidirectional) - Write Arabic text on apps that don't support the bidirectional text.

Reviewer 2: Please only add one app per PR.

> - [Bidirectional](https://github.com/samirdjelal/bidirectional) - Write Arabic text on apps that don't support the bidirectional text.

Reviewer: ```suggestion
- [Bidirectional](https://github.com/samirdjelal/bidirectional) - Write Arabic text in apps that don't support bidirectional text.
```
